gapic-common/gapic-common.gemspec

```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "googleapis-common-protos", ">= 1.3.9", "< 2.0"
   spec.add_dependency "googleapis-common-protos-types", ">= 1.0.4", "< 2.0"
   spec.add_dependency "googleauth", ">= 0.6.2", "< 0.10.0"
-  spec.add_dependency "grpc", ">= 1.7.2", "< 2.0"
+  spec.add_dependency "grpc", "1.24.0.pre1"
```

Comment:
I assume we'll go back to the range when it's no longer a pre-release? Do you have an ETA?

Reply:
It usually takes a couple weeks for grpc to go from pre to actual release. By the end of the month?